src/components/Navbar.jsx

```diff
@@ -0,0 +1,88 @@
+import FUSION from '../assets/FUSION.png'
+import { useNavigate } from 'react-router-dom';
+
+
+function renderTitle(title,subtitle,homeNav){
+    if(homeNav){
+        return(
+        <div className="justify-center content-center text-center justify-items-center space-y-12 p-24">
+            <div className="text-2xl ">
+                EST. 2001
+            </div>
+            <div className="text-8xl">
+                WELCOME TO
+                <div className="font-bold">
+                    FUSION
+                </div>
+            </div>
+            {/* Not sure where to route this to yet, just useless for now */}
+            <button className="bg-[#fac80a] hover:bg-yellow-500 text-black font-bold py-2 px-6">
+                LEARN MORE
+            </button>
+        </div>
+        );
+    } else {
+        return(
+        <div className="justify-center content-center text-center justify-items-center space-y-12 p-24">
+            <div className="text-8xl font-thin">
+                {title}
+            </div>
+            <div className="text-2xl ">
+                {subtitle}
+            </div>
+        </div>
+        );
+    }
+}
+
+export function Navbar({bg, title, subtitle, homeNav = false}){
```

Comment:
ensure to validate the props 

https://legacy.reactjs.org/docs/typechecking-with-proptypes.html